hs_communities/views/communities.py

```diff
@@ -21,41 +26,131 @@ class CommunityView(TemplateView):
     template_name = "hs_communities/community.html"
 
     def dispatch(self, *args, **kwargs):
+        self.template_name = "hs_communities/community.html"
```

Comment:
is this necessary given L26?

Reply:
It is not. I have removed the redundancy now.